### PR TITLE
[16.0][OU-ADD] google_account: Migration done

### PR DESCRIPTION
**Pull request by daiduongnguyen-odoo**

### CODE REVIEW

File: openupgrade_scripts/scripts/google_account/16.0.1.0/upgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---Models in module 'google_account'---
+---Fields in module 'google_account'---
+---XML records in module 'google_account'---
+DEL ir.config_parameter: google_account.config_google_redirect_uri (noupdate)
```
Comment (MiquelRForgeFlow) [inline openupgrade_scripts/scripts/google_account/16.0.1.0/upgrade_analysis_work.txt:4]: shouldn't we use delete_records_safely_by_xml_id()?

Comment (daiduongnguyen-odoo) [reply openupgrade_scripts/scripts/google_account/16.0.1.0/upgrade_analysis_work.txt:4]: Oop miss this, i'll update , thanks

Comment (daiduongnguyen-odoo) [reply openupgrade_scripts/scripts/google_account/16.0.1.0/upgrade_analysis_work.txt:4]: Done sir

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**